kernel/tests/integration-test.rs

```diff
@@ -310,6 +312,7 @@ impl<'a> RunnerArgs<'a> {
             nodes: 0,
             cores: 1,
             memory: 1024,
+            pmem: 1024,
```

Comment:
Can we make pmem Option<usize> for now and only add it if it's specifically requested -- just so we can test functionality with and without pmem available and not accidentially do something in the future that will require to use pmem.

Reply:
Updated.